### PR TITLE
Lowers the cost of hypnogrenades by half and adds warnings regarding it's finnickyness

### PR DESCRIPTION
**Pull request by Singul0**
## About The Pull Request
Lowers the cost of hypnogrenades by half and adds warnings regarding it's finnickyness
## Why It's Good For The Game
Hypnotic grenades, with it's steep cost of 12 TC (1TC MORE of that of the **syndicate bomb**(an item that has a far reaching impact and damage to the station as a whole rather than a one use aoe grenade), at it's current state. it's extremely clunky and risky to use. since it's very easy to blurt out your hypnotic phrase too early or too late (remember, without protection. you yourself are suceptible to to it's effect and stun) before someone says something stupid, it doesn't even have to be in-person but through the radio too.

It would make sense for it to be unreliable if it was half of it's obscene price. 6TC~ or somewhere near that range for it to be unpredictable to use (also with proper warnings of it's dangerousness). But as a it's current state. It's just a noobtrap.

The first iteration of hypnoflash in it's original PR, it's intended for it to be unpredictable and unreliable to use, considering it was 6TC to use and buy. But it raises some concerns on it being so cheap as a one-use AoE OHK item (for non-protected victims).

Just one nerf to the original iteration of hypnoflash would suffice for it to be used as a good niche situational item. Being reliable to use at 12TC or unreliable and risky to use at 6TC. One of those two nerfs, not both would make the item balanced and usable. but in this situation where both nerfs are applied is a classic case of double nerfing something to irrelevancy.
This PR makes it so that it's far more easy and less clunky to use hypnotic grenades.
## Changelog
:cl:
balance :Lowers the cost of hypnogrenades by half and adds warnings regarding it's finnickyness
/:cl:

### CODE REVIEW

**Comment by Singul0** [comment]
> I think the fact that you use it to affect large groups of people is _why_ it can easily be subverted by someone saying "Ling maints" at the wrong time.

Is it not intended to be used for groups of people? If you want to limit hypnotizing people to be on an individual basis, the hypnoflah already exists, and doesn't affect you yourself if you don't have protection. By the hypnogrenade being so volatile to use on it's sole purpose is an intended effect. it kinda makes it useless to use

**Comment by the-orange-cow** [comment]
Sunglasses and earmuffs are relatively easy to get to protect yourself, and I think there is something that can block comms in a short range in the tot shop.

**Comment by Jacquerel** [comment]
It is difficult to use because it can convert a room full of people (you can net quite a lot, as recently demonstrated, by using it in conjunction with the crab phone) into pseudo-antagonists. This is why it has a chance of failure.
I'd actually be _happier_ with you making it cheaper than making it more reliable.

I think that if this item said "throw this into a room and it will turn most of the people in there into revs" it would be obviously regarded as a bad idea even for 12tc. That is functionally what this would be, if you could always reliably guarantee that they would definitely hear exactly what they want you to hear without any other prep.

**Comment by Singul0** [comment]
> It is difficult to use because it can convert a room full of people (you can net quite a lot, as recently demonstrated, by using it in conjunction with the crab phone) into pseudo-antagonists. This is why it has a chance of failure. I'd actually be _happier_ with you making it cheaper than making it more reliable.
> 
> I think that if this item said "throw this into a room and it will turn most of the people in there into revs" it would be obviously regarded as a bad idea even for 12tc. That is functionally what this would be, if you could always reliably guarantee that they would definitely hear exactly what they want you to hear without any other prep.

Eh, i'm honestly fine with it being implemented as it was intended to be in it's original design. where it's unreliable and volatile but at a relatively cheap price (6TC). Though that is kind-of a bit cheap. maybe in-line with the hypnoflash 7TC and a warning for it's volatility?

**Comment by MrMelbert** [comment]
image

**Comment by moocowswag** [comment]
Just use it with radar jammer or after sabotaging tcomms to reduce interference. Hypnogrenade has a very high (round altering) pay out at the expense of a very high skill ceiling and no skill floor

**Comment by Profakos** [comment]
Its not just radio interference. I have fond memories of a round when someone uneffected by it said "oh no", setting this as the command for every hypnotized victim. Honestly more fun than the alternative would have been, with the usual of declaring yourself as the captain.

**Comment by Jacquerel** [comment]
I'm not going to pick for you exactly what to do to buff this item as then I could just make the PR myself, but I'm happy with a number of the things you have mentioned that aren't what the existing PR is

**Comment by Profakos** [comment]
> I'm not going to pick for you exactly what to do to buff this item as then I could just make the PR myself, but I'm happy with a number of the things you have mentioned that aren't what the existing PR is

Oh sorry, I should have clarified that I am against this PR because it would make this too easy to use for its potential strength, and would make the situation I described not exist.

**Comment by Jacquerel** [comment]
> > I'm not going to pick for you exactly what to do to buff this item as then I could just make the PR myself, but I'm happy with a number of the things you have mentioned that aren't what the existing PR is
> 
> Oh sorry, I should have clarified that I am against this PR because it would make this too easy to use for its potential strength, and would make the situation I described not exist.

I should have clarified via quoting that I was talking to the PR author 😅

**Comment by SimplySmiley123** [comment]
> Just use it with radio jammer or after sabotaging tcomms to reduce interference. Hypnogrenade has a very high (round altering) pay out at the expense of a very high skill ceiling and no skill floor
> 
> Raising the skill floor would force nerfs to the skill ceiling

The interferance is other people talking with each other.
You wanna use it on a group of people.
Groups of people, do talk with each other more often then not.

**Comment by ATH1909** [comment]
> > Just use it with radio jammer or after sabotaging tcomms to reduce interference. Hypnogrenade has a very high (round altering) pay out at the expense of a very high skill ceiling and no skill floor
> > Raising the skill floor would force nerfs to the skill ceiling
> 
> The interferance is other people talking with each other. You wanna use it on a group of people. Groups of people, do talk with each other more often then not.

being sent into a trance mutes you, although people with ear or eye protection wouldn't be muted.

**Comment by Singul0** [comment]
Removed the automatic hypno phrase system altogether, reduced the cost and added warnings with how finnicky it is

File: code/game/objects/items/grenades/hypno.dm [hunk 1]
```diff
@@ -1,6 +1,7 @@
 /obj/item/grenade/hypnotic
 	name = "flashbang"
-	desc = "A modified flashbang which uses hypnotic flashes and mind-altering soundwaves to induce an instant trance upon detonation."
+	desc = "A modified flashbang which uses hypnotic flashes and mind-altering soundwaves to induce an instant trance upon detonation. \
+		It seems you can set an hypnotic phrase that's uttered when triggered"
```
Comment (Jacquerel) [inline code/game/objects/items/grenades/hypno.dm:4]:
```suggestion
	desc = "A modified flashbang which uses hypnotic flashes and mind-altering soundwaves to induce an instant trance upon detonation."
```

File: code/modules/uplink/uplink_items/device_tools.dm [hunk 2]
```diff
@@ -232,9 +232,11 @@
 
 /datum/uplink_item/device_tools/hypnotic_grenade
 	name = "Hypnotic Grenade"
-	desc = "A modified flashbang grenade able to hypnotize targets. The sound portion of the flashbang causes hallucinations, and will allow the flash to induce a hypnotic trance to viewers."
+	desc = "A modified flashbang grenade able to hypnotize targets. The sound portion of the flashbang causes hallucinations, and will allow the flash to induce a hypnotic trance to viewers. \
+		Be warned however, that this grenade is finnicky to use and utilize. \
+		Ensure proper protection and instant uttering of hypnotic command before triggering the grenade."
```
Comment (Jacquerel) [inline code/modules/uplink/uplink_items/device_tools.dm:237]:
```suggestion
	desc = "A modified flashbang grenade able to hypnotize targets. The sound portion of the flashbang causes hallucinations, and will allow the flash to induce a hypnotic trance to viewers. \
		Equipment to protect yourself from the effects are not provided. Ensure you are ready before pulling the pin."
```
I don't think the wording as you provided it works for the in-character uplink, in fact I think your wording is actively misleading (it sounds like you should say the command before triggering the grenade).

**Review by Jacquerel — CHANGES_REQUESTED** [review]

**Comment by Higgin** [comment]
might want to change the PR title to reflect the shift in direction, but making them cheaper in quantity and putting in a warning are both positive changes

**Comment by XDTM** [comment]
Don't forget the changelog as well

**Comment by Singul0** [comment]
>If those are your issues with it just make it into a box which has earmuffs sunglasses the grenade itself and a voice recorder for easier messaging and maybe a short one use comms jammer.

That just removes all the short quests you usually do to use the item effectively, IMO that just serves the same purpose in my original pr without addressing the problems that lies with said implemention. except it's down from 100% reliable on non-protected targets to 95% and you're also stuck with a surprising amount of leftover tools you get after using the singleuse grenade.

>6 TC is also neat but let me tell you right now it will be too strong. There are ways to use it well, it's just most older players don't bother because stun click with esword till dead. You make it 6 TC and get ready for it to be an annoying mainstay item.
> I sentence you to relax and use your next traitor roll to get a surplus crate instead of the same "efficient" loudout you've been using for years.

chief, the reason i'm pushing this PR so intensely is because of how fine-tuned the meta for traitors are, mostplayers would stick to your regular old surface level mechanic weapons like eswords, guns to murderbone instead of engaging with deeper mechanics and interactions of the game (people usually pick the boring old meta instead of spending time to do gimmicks that's uncertain to work) by making these niche gimmicks gadgets actually bearable to use is one step to decrease that prevalence of boring murderbone meta-gear.

Also, i'm willing to bet it's not going to be a meta-mainstay item. at least not to the point of eswords, revolvers. The meta you usually see are powerful standard weapons and QoL gadgets. These kinds of stuff have never ever been popular to the playerbase (case and point the hypnoflash. a pretty powerful item only being used occassionally). If it does get too much of a meta item. limiting it to 2-3 purchases only is enough of a fix. you gotta remember that this shit costs a lot for a single use items. whereas most other items at traitors disposal are tools that can be used multiple times

**Comment by Dawnseer** [comment]
Because if there's anything we've learned from recent discussions, it's that not enough hypnosis is going on.

**Comment by optimumtact** [comment]
this change smells like salt to me.

**Comment by Singul0** [comment]
Eh, it wasn't oh well

**Pull request closed without merging**